ansible/docker_site.yml

```diff
@@ -3,8 +3,7 @@
 
 - hosts: all
   vars:
-    ros_release: "noetic"
-    ansible_python_interpreter: /usr/bin/python3
+    ros_release: "indigo"
```

Comment:
How come this has been set to indigo?

Reply:
I didn't do this myself, must of been an issue when switching branches from master->python3->SRC-5241, will alter!